File: InterviewQuestions/Python Interview Programming Questions/task8.py
# Write a Python function that returns the Fibonacci sequence up to a given number of terms


def FibonacciSeries(count):

    number1 = 0
    number2 = 1

    iteration = 0

    while iteration < count:

        if iteration == 0:

            print(number1)
            print(number2)

            iteration = iteration + 2
        
        else:

            total = number1 + number2

            print(total)

            number1 = number2
            number2 = total

            iteration = iteration + 1

FibonacciSeries(10)

File: InterviewQuestions/Javascript Interview Programming Questions/task8.js
// Write a JavaScript function that returns the Fibonacci sequence up to a given number of terms


const FibonacciSeries = count => {

    let number1 = 0
    let number2 = 1

    let iteration = 0

    while (iteration < count) {

        if (iteration === 0) {

            console.log(number1)
            console.log(number2)
            iteration = iteration + 2
        }
        else {
            
            sum = number1 + number2
            console.log(sum)

            number1 = number2
            number2 = sum

            iteration = iteration + 1
        }
    }
}

FibonacciSeries(10)